pimd/pim_util.c

```diff
@@ -126,20 +128,90 @@ int pim_is_group_224_4(struct in_addr group_addr)
 	return prefix_match(&group_all, &group);
 }
 
-bool pim_is_group_filtered(struct pim_interface *pim_ifp, pim_addr *grp)
+static bool pim_cisco_match(const struct filter *filter, const struct in_addr *source,
```

Comment:
Why isn't this commit broken up into multiple commits?  When I see `oh and I moved a bunch of stuff around` in the commit message and only see one commit it makes it much harder to review the changes for functionality -vs- the changes for structure.  Additionally it's hard to review and figure out what is going on.